pkg/cmd/server/start/start_node.go

```diff
@@ -225,17 +226,17 @@ func (o NodeOptions) CreateNodeConfig() error {
 	return nil
 }
 
-func RunSDNController(config configapi.NodeConfig) {
+func RunSDNController(config configapi.NodeConfig, nodeConfig *kubernetes.NodeConfig) {
```

Comment:
also, best arg names ever :)
use the config. the node config? no, the node config.